### PR TITLE
Append get_asset_details_command

### PR DESCRIPTION
**Pull request by inbalapt1**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-11804)
## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Core/Integrations/CortexCoreIR/README.md [hunk 1]
```diff
@@ -2897,3 +2897,78 @@ Stops a process by its causality ID.
     ]
 }
 ```
+
+### core-get-asset-details
+
+***
+Get UAI asset info
```
Comment (omerKarkKatz) [inline Packs/Core/Integrations/CortexCoreIR/README.md:2904]: What is UAI ?

**Review by omerKarkKatz — APPROVED** [review]
Nice job!

**Comment by kobymeir** [comment]
### This PR was automatically updated by a [GitHub Action](https://github.com/demisto/content/actions/runs/12219590859)
- **Core** pack version was bumped to **3.2.0**.

To stop automatic version bumps, add the `ignore-auto-bump-version` label to the github PR.

File: Packs/Core/Integrations/CortexCoreIR/CortexCoreIR.yml [hunk 2]
```diff
@@ -3094,11 +3094,56 @@ script:
     - contextPath: Core.TerminateCausality.action_id
       description: The action id.
       type: String 
+  - arguments:
+    - description: asset unique identifier.
```
Comment (ShirleyDenkberg) [inline Packs/Core/Integrations/CortexCoreIR/CortexCoreIR.yml:3098]:
```suggestion
    - description: Asset unique identifier.
```

File: Packs/Core/Integrations/CortexCoreIR/README.md (hunk 1 above)
Comment (ShirleyDenkberg) [inline Packs/Core/Integrations/CortexCoreIR/README.md:2904]:
```suggestion
Get UAI asset information.
```

File: Packs/Core/Integrations/CortexCoreIR/README.md [hunk 3]
```diff
@@ -2897,3 +2897,78 @@ Stops a process by its causality ID.
...
+
+#### Base Command
+
+`core-get-asset-details`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| asset_id | asset unique identifier. | Required |
```
Comment (ShirleyDenkberg) [inline Packs/Core/Integrations/CortexCoreIR/README.md:2914]:
```suggestion
| asset_id | Asset unique identifier. | Required | 
```

File: Packs/Core/ReleaseNotes/3_2_0.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Investigation & Response
+
+- Added the *core-get-asset-details* command.
```
Comment (ShirleyDenkberg) [inline Packs/Core/ReleaseNotes/3_2_0.md:6]:
```suggestion
- Added the ***core-get-asset-details*** command.
```

**Comment by ShirleyDenkberg** [comment]
@omerKarkKatz @maimorag Doc review completed.

**Comment by kobymeir** [comment]
### This PR was automatically updated by a [GitHub Action](https://github.com/demisto/content/actions/runs/12221532525)
- **Core** pack version was bumped to **3.3.0**.

To stop automatic version bumps, add the `ignore-auto-bump-version` label to the github PR.

**Pull request merged**